collectors/nvd/collectors.py

```diff
@@ -251,6 +253,13 @@ def collect(self, cve: Union[str, None] = None) -> str:
             # create a new snippet with a flaw (if it does not exist) and link them
             snippet = None
             if self.snippet_creation_enabled:
+                # we do not care about historical data
+                published = timezone.make_aware(
+                    dateparse.parse_datetime(item["published_in_nvd"])
+                )
+                if published < SNIPPET_CREATION_START_DATE:
```

Comment:
I would be for an option to leave that constant empty (no environment variable set) meaning that we collect everything. Otherwise if you really want everything you have the only option to set it really in history but that sounds not very convenient.